docs/theming.md

```diff
@@ -430,37 +430,35 @@ Since our cards only contain text and no components, let's keep it simple for no
 > Note: in the future we plan to provide a Javascript utility method for changing all derived colors and making this
 use-case easier.
 
+## Custom Sass Mixins
 
-## Dark Themes
+We also have provided a way to customize on a per component basis. We realize that you may want a button with a
```

Comment:
I think it would be okay to focus on a single example here, and use `mdc-button-filled-accessible` which automatically picks ink color based on fill color for best contrast.

Lead off with something like, "For example, let's say you want to make particular buttons stand out by changing their fill color."